### PR TITLE
bots: Enable lorax cloud tests

### PR DESCRIPTION
**Pull request by atodorov**
Note: for RHEL images I've used `rhel-8` and `rhel-7` in the hope that they will always point to the latest minor version so we don't have to update them on every release. Let me know if such images don't exist and I will update the PR.

### CODE REVIEW

**Comment by atodorov** [comment]
CC @martinpitt @larskarlitski @jikortus

File: bots/tests-scan [hunk 1]
```diff
@@ -78,7 +78,14 @@ REPO_BRANCH_CONTEXT = {
             'cockpit/fedora-30/vmware',
         ],
         '_manual': [
-            'cockpit/rhel-8-0',
+            'cockpit/rhel-8',
```
Comment (martinpitt) [inline bots/tests-scan]: There is no "rhel-8" image indeed, it needs to be explicitly "rhel-8-0" or "rhel-8-1".

File: bots/tests-scan [hunk 2]
```diff
@@ -86,7 +86,13 @@ REPO_BRANCH_CONTEXT = {
             'cockpit/rhel-8/openstack',
             'cockpit/rhel-8/vmware',
 
-            'cockpit/rhel-8-1',
+            'cockpit/rhel-7',
```
Comment (martinpitt) [inline bots/tests-scan]: Similarly, this needs to be "rhel-7-7".

File: bots/tests-scan (hunk 1 above)
Comment (atodorov) [reply bots/tests-scan]: Updated and filed https://github.com/cockpit-project/cockpit/issues/12031 to discuss resolving this.

**Comment by atodorov** [comment]
> Thanks! I force-pushed to add the "Closes #" to the commit message instead of the Subject:

FTR I've had this updated when you asked me to so you shoudn't have had to do anything :-)

**Pull request closed without merging**